rules/windows/credential_access_domain_backup_dpapi_private_keys.toml

```
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/08/13"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/08/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies creation of Domain Backup private keys, which may indicate activity related to Domain Backdup DPAPI
```

```suggestion
Identifies creation or modification of Domain Backup private keys, which may indicate activity related to Domain Backdup DPAPI
```

because this would apply to `event.type:(creation or change)` as currently written